src/writeData/subscriptions/components/BrokerFormContent.tsx

```diff
@@ -185,7 +186,7 @@ const BrokerFormContent: FC<Props> = ({
                 handleValidation('Broker Host', formContent.brokerHost)
               }
               helpText={
-                edit
+                className !== 'create' && edit
```

Comment:
Seems like you're deriving logic here from a composed value. If the state isn't entirely connected to styling / class name, the source value should be pushed into the component instead. Frees you up to add a nested BEM while still keeping the text change logic.